generator/template/model_template_test.go

```diff
@@ -43,3 +45,29 @@ func Test_TableModelField(t *testing.T) {
 		Tags: nil,
 	})
 }
+
+func Test_Model_ShouldSkip(t *testing.T) {
```

Comment:
This test doesn't add much value, as it just sets a field and then reads the same value that was assigned.
Take a look at [generator_test.go](https://github.com/go-jet/jet/blob/master/tests/postgres/generator_test.go).

Reply:
Will remove, agreed that they add very little value. Added these since the gocov check was failing after the patch.

Reply:
> Added these since the codecov check was failing after the patch.

I see. In that case new test in [generator_template_test.go](https://github.com/go-jet/jet/blob/master/tests/postgres/generator_template_test.go) would be a better option, since `generator_test.go` is testing jet generator executable and thus codecov is not included in the result.

Reply:
The functionality I'm introducing is already covered by existing tests in `generator_template_test.go`, since I'm just externalizing the `Skip` boolean to the CLI. For example:

https://github.com/go-jet/jet/blob/6c1224017dcdcd54036da3d80902d5aa996eaa19/tests/postgres/generator_template_test.go#L58-L73

Happy to add a test to [generator_test.go](https://github.com/go-jet/jet/blob/6c1224017dcdcd54036da3d80902d5aa996eaa19/tests/postgres/generator_test.go), if you'd like, despite it not being included in codecov.

Reply:
Updating any existing tests in `generator_template_test.go` to include `ShouldSkip` method call should be enough for codecov to pass.